content/pages/amp-roadshow.html

```diff
@@ -0,0 +1,204 @@
+---
+$title: AMP Roadshow 2017/2018
+noglobalnote: 1
+class: amp-roadshow
+
+components:
+  - iframe
+stylesheet: pages/amp-roadshow.css
+
+$localization:
+    locales:
+    - en
+---
+
+{% set conf = g.doc('/content/includes/amp-roadshow.yaml', locale=doc.locale) %}
+
+<section class="teaser amp-roadshow-section">
+
+    <div class="wrapper top">
+
+        <div class="left">
+            <p>Join the worldwide</p>
+            <h2>AMP Roadshow</h2>
+            <ul>
+                <li><a href="#cities">Cities</a></li>
+                <li><a href="#schedule">Agenda</a></li>
+                <li><a href="#speakers">Content</a></li>
+                <li><a href="#workshops">Codelabs</a></li>
+                <li><a href="#faq">FAQ</a></li>
+            </ul>
+        </div>
+
+        <div class="right box">
+            <p><strong>We've got plenty of new stuff to share.</strong></p>
+
+            <p>Come join us in your city for a free, inclusive, full-day dev workshop with the AMP team.</p>
+
+            <p class="next"><span class="icon-plane"></span>Next stop: <a href="{{ conf.next.link }}">{{ conf.next.title }}</a></p>
```

Comment:
There wasn't anything populated for "next stop" when I ran it locally, should it?